hikey.mk

```diff
@@ -209,7 +209,6 @@ xtest: xtest-common
 # "make clean" in xtest: fails if optee_os has been cleaned previously
 .PHONY: xtest-clean
 xtest-clean: xtest-clean-common
-	rm -rf $(OPTEE_TEST_OUT_PATH)
```

Comment:
I think it's better to change to:
```
rmdir --ignore-fail-on-non-empty $(OPTEE_TEST_OUT_PATH)
```
It's going to handle both cases: remove empty dir, or leave it in case if it's not empty(in case OPTEE_TEST_OUT_PATH is redefined to some critical path/patterns) 
What do you think?